Update dot GitHub (#103)
Daimler/daimler-foss: README.md

> # Mercedes-Benz Group FOSS Materials

Reviewer: not sure if "materials" is a "Germanism" here; my sense of English would rather go with "Information".

Author: Thanks for the hint. I replaced "materials" by "documents".

> This organization on github.com is our primary location to publish open source software projects and to host documents that support this endeavour like our [FOSS code of conduct](https://github.com/mercedes-benz/.github/blob/main/CODE_OF_CONDUCT.md) and [security policy](SECURITY.md). Find more supporting documents in our [profile repo](https://github.com/mercedes-benz/.github).

Reviewer 2: Maybe we should reference the org explicitely instead of using "this organization" to make it more clear. Furthermore I would point the SECURITY.md directly to the one in the `.github` repository in the same way as the `CODE_OF_CONDUCT.md`.

```suggestion
The [mercedes-benz](https://github.com/mercedes-benz) organization on github.com is our primary location to publish open source software projects and to host documents that support this endeavour like our [FOSS code of conduct](https://github.com/mercedes-benz/.github/blob/main/CODE_OF_CONDUCT.md) and [security policy](https://github.com/mercedes-benz/.github/blob/main/SECURITY.md). Find more supporting documents in our [profile repo](https://github.com/mercedes-benz/.github).
```

In this repo we mainly provide provide a list of custom license identifiers for our FOSS disclosure process.

# Contributing

> Despite the public nature of this repo, we would rather that you do not contribute here but rather look forward to your contributions on our software projects since this repo focusses on supporting materials that support our open source journey.

Reviewer: the current version reads very bumpily IMO:
```suggestion
Despite the public nature of this repository, we prefer you not contributing here but rather look forward to your contributions to our software projects since this repository focuses on materials supporting our open source journey.
```

# Contact

> We really want to learn from you and are happy to hear your thoughts and questions at <foss@mercedes-benz.com>.

Reviewer: is that e-mail address still active or has it been transformed into a forward for our serviceNow instance?

Author: ServiceNow entry is only for internals. For externals, this email is still valid. FOSS Office will turn such incoming emails into ServiceNow tickets.
